src/modules/enrollment/components/pages/EnrollmentDashboard.tsx

```diff
@@ -112,6 +113,16 @@ const EnrollmentDashboard: React.FC = () => {
       focusMode={focusMode}
     >
       <Container maxWidth='xl' disableGutters>
+        <LocalVersionCoordinationPrompt
+          recordType='Enrollment'
+          recordId={enrollment.id}
+          currentVersion={enrollment.lockVersion}
+        />
+        <LocalVersionCoordinationPrompt
+          recordType='Client'
+          recordId={client.id}
+          currentVersion={client.lockVersion}
+        />
```

Comment:
Could end up with overlapping modals here. We could refactor the LocalVersionCoordinationPrompt to handle that

Reply:
this doesn't seem to be an issue in practice